docs/site/todo-tutorial-datasource.md

```diff
@@ -67,6 +67,13 @@ Create a `data` folder in the applications root and add a new file called
 }
 ```
 
+> **NOTE**:
```

Comment:
I think this should be removed, now that you are using `{% include note.html ... %}`